fix: better error message when languages do not have a builtin QP

gtoison opened the pull request:
Hello,
As noticed [here](https://github.com/spotbugs/sonar-findbugs/issues/1271#issuecomment-2931463681) the error message for missing builtin quality profiles is a bit confusing:
```
The following languages have no built-in quality profiles: kubernetescssscalajsppyjsansibledockerplsqlrustdartjavawebflexxmljsonipynbtextvbnetcloudformationgrvyyamlswiftcppcgokotlinneutraltsqlsecretsrubycsshellphpterraformazureresourcemanagerabapobjcts
```

dejan-milisavljevic-sonarsource commented:
Can we have a unit test to ensure the format is now correct and will remain correct?

gtoison commented:
Thanks for reviewing @OrlovAlexander85 
@dejan-milisavljevic-sonarsource I have added a unit test to check that the message looks right and I have rebased the branch on the master.

gtoison commented:
Cool, thanks!
I guess the potential problem was that `BuiltInQProfileRepositoryImpl` has two constructors (one without the quality profiles), so when the quality profiles fail to load for some reason, we end up in a situation where Spring initializes it without any quality profile.
This was discovered here: https://github.com/spotbugs/sonar-findbugs/issues/1271#issuecomment-2931463681

This strange error message (not built-in QP for any of the installed languages) made it hard to realize that _one_ quality profile from the plugin had missing dependencies and could not initialize.
